scripts/docker-tcp.socket

```diff
@@ -1,10 +0,0 @@
-[Unit]
```

Comment:
I hope this wont cause trouble in out docker swarm setup. When you do `make demo` it brings up 3 node docker swarm cluster.